R/r1_opt.R

```diff
@@ -105,9 +114,8 @@ You could ignore this warning, but you might like to check out https://github.co
     }
   }
 
-  return(list(l=l,f=f,l2=l2,f2=f2,tau=tau,F_obj=F_obj,KL_l = KL_l, KL_f = KL_f,
-              gl = ebnm_l$fitted_g, gf = ebnm_f$fitted_g,
-              penloglik_l = ebnm_l$penloglik, penloglik_f = ebnm_f$penloglik,
+  return(list(l=l, f=f, l2=l2, f2=f2, tau=tau, F_obj=F_obj, KL_l=KL_l, KL_f=KL_f,
```

Comment:
don't you need to set 
``gl = ebnm_l$fitted_g, gf = ebnm_f$fitted_g, penloglik_l = ebnm_l$penloglik, penloglik_f = ebnm_f$penloglik``
before this? I guess i don't see where gl etc are being set (except to NULL at initialization)